dvc/annotations.py

```diff
@@ -10,11 +10,13 @@ class Annotation:
     PARAM_TYPE: ClassVar[str] = "type"
     PARAM_LABELS: ClassVar[str] = "labels"
     PARAM_META: ClassVar[str] = "meta"
+    # PARAM_PATH: ClassVar[str] = "path"
```

Comment:
This needs to be introduced, making current annotation and the one from `artifacts:` section different. Will be working around this

Reply:
There is already a `path` field in `.dvc` files:

```yaml
md5: aa562ba5bf3d8b25eef7167aeb102dc0
frozen: true
deps:
- path: get-started/data.xml
  repo:
    url: https://github.com/iterative/dataset-registry
    rev_lock: cf6481baf56f156aa0876709cc231aaf3f3a3c29
    rev: get-started-40K
outs:
- md5: 22a1a2931c8370d3aeedd7183606fd7f
  size: 14445097
  path: data.xml
  desc: Imported raw data (tracks source updates)
```

Is a new field needed? Does it cause any conflicts?